splunk-otel-android/src/main/java/com/splunk/rum/RumInitializer.java

```diff
@@ -248,30 +241,20 @@ private void installLifecycleInstrumentations(
                 });
     }
 
-    private Resource createResource() {
+    private Resource createSplunkResource() {
         // applicationName can't be null at this stage
         String applicationName = requireNonNull(builder.applicationName);
         ResourceBuilder resourceBuilder =
-                Resource.getDefault().toBuilder()
-                        .put(APP_NAME_KEY, applicationName)
-                        .put(SERVICE_NAME, applicationName);
+                Resource.getDefault().toBuilder().put(APP_NAME_KEY, applicationName);
```

Comment:
Should we override the default `SERVICE_NAME` with `applicationName`? To keep them consistent

As a separate story/improvement, I guess we could make it optional and allow not specifying it in the `SplunkRumBuilder`

Reply:
Yeah, but I didn't want to keep that in the splunk side of things -- the setting of `SERVICE_NAME` with the app name is happening over in the `AndroidResource`. I thought it was important enough to elevate.